std/conv.d

```diff
@@ -1738,6 +1754,18 @@ T toImpl(T, S)(S value)
     return parse!T(value);
 }
 
+T toImpl(T, S)(S value, AllowUnderscores uscores)
```

Comment:
I think you need similar template constraints as above (e.g. `isDynamicArray!S`)